src/PageLayout/Notification/Details/RequestSummary.js

```diff
@@ -0,0 +1,86 @@
+                            width={'350px'}
+                            marginRight={'30px'}
+                            marginTop={'0px'}
+                            marginBottom={'0px'}
+                            marginLeft={'10px'}
+                            className="map"></Map>
+                    </div>
+                </div>
+                { /* View for DG*/
+                    this.props.details.receivingAgency.claimed &&
+                        <div className="flex">
+                            <div className="receiving-agency">
+                                <h2>Recipient</h2>
+                                <p className="donating-recieving" id="donatingAgency">{this.props.details.raInfo.name}</p>
+                                <p className="donating-recieving" id="receivingContactName">{this.props.details.raInfo.primaryContact.name}</p>
+                                <p className="donating-recieving" id="receivingContactPhone">{this.props.details.raInfo.primaryContact.phone}</p>
+                                <p className="donating-recieving" id="receivingContactEmail">{this.props.details.raInfo.primaryContact.email}</p>
+                            </div>
+                            <div className="notification-map">
+                                <Map 
+                                    address={this.props.details.raInfo.address} 
+                                    height={'150px'}
+                                    width={'350px'}
+                                    marginRight={'30px'}
+                                    marginTop={'0px'}
+                                    marginBottom={'0px'}
+                                    marginLeft={'10px'}
+                                    className="map"></Map>            
+                            </div> 
+                        </div>
+                }
+                {/* View for RA */
+                    this.props.details.delivererGroup.claimed  &&
+                        <div className="receiving-agency">
+                            <h2>Deliverers</h2>
+                            <div className="modal-deliverers">
+                                {
+                                    this.props.details.dgInfo.deliverers.length === 0 ?
+                                        <p>To be determined</p>
+                                        :
+                                        this.props.details.dgInfo.deliverers.map((d, i) => {
+                                            return (
+                                                <div className="mobile-student-info" id={i} key={i}>
+                                                    <p id="mobile-name">{d.name}</p>
+                                                    <p id="mobile-org">{d.email}</p>
+                                                    <p id="mobile-org">{d.phone}</p>
+                                                </div>
```

Comment:
actually, just realized, this section for DG info is actually supposed to show the same thing as DA & RA except the map. So: agency name, primary contact's name/phone/email. Not the deliverers (it's actually impossible for a request to have deliverers during this notification flow).